Dockerfile

```diff
@@ -4,7 +4,7 @@ ARG VERSION=""
 ARG BUILDNUM=""
 
 # Build Geth in a stock Go builder container
-FROM golang:1.23.4-alpine@sha256:20c0c5ccf7c851ecde9f0664404213adabe2b47f34e5292bfbeaca203241836c as builder
+FROM golang:1.24-alpine AS builder
```

Comment:
Was this an upstream change? - If possible, it would be nice to keep this pinned to a SHA256 digest (and for the other Docker images).